### PR TITLE
Clarify that osu!mania hitsounding exemptions are for hitsound additions

### PR DESCRIPTION
**Pull request by cl8n**
becomes a bit repetitive with the footnote but [someone asked for clarity here](https://osu.ppy.sh/community/forums/posts/8670427) and doesn't do any harm. made some tiny fixes to the topic in related article

### CODE REVIEW

File: wiki/Ranking_Criteria/en.md [hunk 1]
```diff
@@ -199,8 +199,8 @@ This category contains explicit allowance statements of concepts and rules that
...
   - **...not be encoded upwards from a lower bitrate.**
 - **A beatmap may only contain one song file used by all difficulties.** Multiple song files within a single beatmap are unsupported and result in unexpected behaviour with preview times, metadata, etc.
-- **Beatmaps must be [hitsounded](/wiki/Beatmapping/Hitsound).** Hitnormals give feedback to the player, and additions (whistles, claps, and finishes) accent the most important parts of the music.
-  - **osu!mania beatmaps do not require hitsound additions.** This is to allow for easier approachability to osu!mania mappers of different upbringings. It is still highly recommended to add hitsounds to improve the feel of your beatmaps.
+- **Beatmaps must be [hitsounded](/wiki/Beatmapping/Hitsound).** Hitnormals give feedback to the player, and additions (whistles, claps, and finishes) accent the most important parts of the music.[^normal-vs-addition]
+  - **osu!mania beatmaps do not require hitsound additions[^normal-vs-addition].** This is to allow for easier approachability to osu!mania mappers of different upbringings. It is still highly recommended to use hitsound additions to improve the feel of your beatmaps.
```
Comment (Walavouchey) [inline wiki/Ranking_Criteria/en.md:203]: these come after punctuation

```suggestion
  - **osu!mania beatmaps do not require hitsound additions.**[^normal-vs-addition] This is to allow for easier approachability to osu!mania mappers of different upbringings. It is still highly recommended to use hitsound additions to improve the feel of your beatmaps.
```

File: wiki/Beatmapping/Hitsound/en.md [hunk 2]
```diff
@@ -13,7 +19,7 @@ tags:
 
 **Hitsounds** are the sounds that osu! plays in response to user input when interacting with [hit objects](/wiki/Hit_object). Hitsounds are typically used to give auditory feedback to the player to help them judge their [accuracy](/wiki/Gameplay/Accuracy) in relation to the song.
 
-Hitsound feedback consists of a default sample called a "hit normal", and any combination of whistle, finish, or clap sample additions. Each sample has one of three "sample sets" that change its style: `Normal`, `Soft`, or `Drum`.
+Hitsound feedback consists of a default sample called a *hitnormal*, and any combination of whistle, finish, or clap sample *additions*. Each sample has one of three "sample sets" that change its style: `Normal`, `Soft`, or `Drum`.
```
Comment (Walavouchey) [inline wiki/Beatmapping/Hitsound/en.md:22]:
```suggestion
Hitsound feedback consists of a default sample called a *hitnormal*, and any combination of whistle, finish, or clap sample *additions*. Each sample is part of one of three *sample sets* that denote different styles: `Normal`, `Soft`, or `Drum`.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Ranking_Criteria/en.md (hunk 1 above)
Comment (cl8n) [reply wiki/Ranking_Criteria/en.md:203]: i put this one inside cuz its just for "hitsound additions", not whole sentence

File: wiki/Beatmapping/Hitsound/en.md (hunk 2 above)
Comment (cl8n) [reply wiki/Beatmapping/Hitsound/en.md]: not even related to pr but :ok:  :100:

File: wiki/Ranking_Criteria/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Ranking_Criteria/en.md:203]: they come after punctuation anyway, with very few exceptions. we're following [wikipedia](https://en.m.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Punctuation_and_footnotes) on this. see https://github.com/ppy/osu-wiki/pull/7391

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**